recipe/meta.yaml

```diff
@@ -41,6 +36,7 @@ requirements:
     - cuda-version {{ cuda_compiler_version }}  # same cuda for host and build
     - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
     - libcublas-dev    # [(cuda_compiler_version or "").startswith("12")]
+    - libcurand-dev    # [(cuda_compiler_version or "").startswith("12")]
     - libcusolver-dev  # [(cuda_compiler_version or "").startswith("12")]
     - libcusparse-dev  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
Note that all of these CUDA packages were here before to satisfy PyTorch's header requirements. The only new on is `libcurand-dev`. Perhaps this comes up as some of the new extensions use other bits from PyTorch that were not used before